packages/firebase_crashlytics/example/android/build.gradle

```diff
@@ -2,15 +2,12 @@ buildscript {
     repositories {
         google()
         jcenter()
-        maven {
-            url 'https://maven.fabric.io/public'
-        }
     }
 
     dependencies {
         classpath 'com.android.tools.build:gradle:3.3.2'
         classpath 'com.google.gms:google-services:4.3.0'
-        classpath 'io.fabric.tools:gradle:1.26.1'
+        classpath 'com.google.firebase:firebase-crashlytics-gradle:2.0.0-beta03'
```

Comment:
Looks like they have moved to **beta4** as per the official doco [here](https://firebase.google.com/docs/crashlytics/upgrade-sdk?platform=android)